sdk-setup/src/mb2

```diff
@@ -351,7 +354,7 @@ COMMANDS
         effect is cumulative and persists until overriden with later name
         prefixed with the opposite operator.
 
-    deploy {--pkcon|--rsync|--sdk|--zypper|--manual} [--all] [--debug]
+    deploy {--pkcon|--rsync|--sdk|--zypper|--manual} [--all] [--debug] [--nosign]
```

Comment:
As we discussed before, it should be changed so that `deploy` never implies `package` (currently it does behind qtc only). When this is done, no signing option (--sign/--nosing) should be needed here.

Reply:
May be i miss something, but how it's work?
In `MerMb2RpmDeployStep` call only mb2 deploy and in each time it rebuild rpm packages.

Reply:
Here you are changing the deploy configuration so that it does building and deploying RPM packages with separate steps https://github.com/sailfishos/sailfish-qtcreator/pull/325/commits/557dbb18645bcaaf7e985a8245f5caf3215ba253 .

And here https://github.com/sailfishos/sdk-setup/pull/179#discussion_r446943892 I suggest that instead of achieving that by adding `--deploy-only` we should stop implying run_package with run_deploy.

So since the deploy commands does not rebuild the packages, it does not need the signing option.

Reply:
Oh, yes, sorry, updated deploy command according new deploy mechanism